Nos/Service/Relay/Filter.swift

```diff
@@ -5,13 +5,13 @@ import Foundation
 struct Filter: Hashable, Identifiable {
     
     /// List of author identifiers the Filter should be constrained to.
-    let authorKeys: [RawAuthorID]
+    var authorKeys: [RawAuthorID]
```

Comment:
Unless I'm missing something, `authorKeys` and `kinds` (below) can still be immutable.
```suggestion
    let authorKeys: [RawAuthorID]
```